django_freeradius/api/views.py

```diff
@@ -75,9 +79,7 @@ class AccountingView(generics.ListCreateAPIView):
     """
     HEADER: Pagination is provided using a Link header
             https://developer.github.com/v3/guides/traversing-with-pagination/
-
     GET: get list of accounting objects
-
```

Comment:
don't touch other parts of the code that are not related to your patch please, it's a bad practice